embulk-core/src/main/java/org/embulk/spi/util/Timestamps.java

```diff
@@ -30,6 +31,20 @@ private Timestamps() {}
         return parsers;
     }
 
+    public static Map<Column, TimestampParser> newTimestampColumnParsersAsMap(
```

Comment:
Sorry that it's really an internal and implicit reason, but actually, we basically don't want to expand existing "library" methods and classes anymore, especially under `spi.util`. (We'll eventually be extracting such utility/library classes as a separate artifact, not in embulk-core.) Once we have it under `spi.util`, plugin developers start using this, and we won't be able to remove it for a long time.

Then, if it can be a private method on `JsonParserPlugin` side, let's have it there.

(And, possibly nice to have that as a TODO comment.)

Reply:
OK, moved it to the inside of the plugin in f45faac